tests/lib/options_helpers.py

```diff
@@ -26,8 +24,10 @@ class AddFakeCommandMixin(object):
 
     def setup(self):
         self.environ_before = os.environ.copy()
-        commands_dict[FakeCommand.name] = FakeCommand
+        commands_dict['fake'] = (
+            'tests.lib.options_helpers', 'FakeCommand', 'fake summary',
```

Comment:
Ah! This would be complicated by my suggestion for simplification and DRYing of "pip._internal.commands.". Let's look into that in a follow up then. :)